news/2024/2024-11-28-osu-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,135 @@
+
+Expectations for this match were sky-high, and it lived up to the billing, a back-and-forth affair in which the teams dominated their own picks to a historic level. The United States, having bolstered their roster's well-roundedness with the return of ::{ flag=US }:: [Utami](https://osu.ppy.sh/users/7512553) and the addition of rookie ::{ flag=US }:: [Pezz](https://osu.ppy.sh/users/10651106), opted heavily into the gimmicky maps in the pool, opening things up with the overlap-reading No Mod pick [Lovely Icecream Princess Sweetie](https://osu.ppy.sh/beatmapsets/669232#osu/1415526) before delving into the precision flow aim hard rock pick [Saika](https://osu.ppy.sh/beatmapsets/1224300#osu/2546419). They would win both, with ::{ flag=US }:: [Pezz](https://osu.ppy.sh/users/10651106) and ::{ flag=US }:: [Utami](https://osu.ppy.sh/users/7512553) their top two performers.
+
+Not to be outdone, mechanics marvels Australia simply decided to pick their way straight through the double time pool, and the Aussies would be rewarded with wins on all four DT maps, including a ridiculous 4.39 million team score on [Hit & Run](https://osu.ppy.sh/beatmapsets/2281450#osu/4864105) (DT4). Back and forth went the scoreline as neither team could pick up a break point through the first eight maps of the match, bringing the score to 4–4 heading into the teams' respective final picks.
+
+After the US won their last map, the last pick of the match belonged to Australia. If they won, it would force a tiebreaker; lose, and the match would end right then and there with an American victory. Out of double time maps, they opted into the aim consistency No Mod selection of [Warcry](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391), an incredibly popular pick the entire weekend. To this point, ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) had been having something of a quiet match by his illustrious standards; while he put up several full combos and top scores, none truly stood out in the way many expected — as one of the pre-eminent aim players in the tournament, what would NM1 bring for him?
+
+Unfortunately, the answer was despair. After an early break, he was regaining combo when the lobby came to a crashing halt. It was a disconnect from the world #1, his internet having dropped at the worst possible time. The map continued on for the seven other players — unaware of what had occurred, the United States eventually won by a margin of 844,321. ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) would not have been able to make up the difference in score, so by rule the United States won the pick and with it the match.
+
+It was a disappointing ending to a tremendous match that saw two top teams at their absolute peak, with the United States moving on to their first winners bracket final since 2021 while Australia fell to the losers bracket to fight their way through.
```

Comment:
```suggestion
It was a disappointing ending to a tremendous match that saw two top teams at their absolute peak, with the United States moving on to their first winners' bracket final since 2021, while Australia fell to the losers' bracket to fight their way through.
```